.github/workflows/publish-latest.yml

```
@@ -0,0 +1,29 @@
+name: Publish to @next
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    environment: ci
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Setup Node.js LTS
+        uses: actions/setup-node@v2
+        with:
+          node-version: lts/*
+          cache: 'yarn'
+      - name: Install packages
+        run: yarn --frozen-lockfile
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        uses: changesets/action@b98cec9
```

Last adjustment, I'm using full-length sha here.
```suggestion
        uses: changesets/action@b98cec97583b917ff1dc6179dd4d230d3e439894
```